shared/actions/tracker.js

```diff
@@ -411,11 +416,11 @@ function updatePGPKey (username: string, pgpFingerprint: Buffer): Action {
   }
 }
 
+const sessionIDToUsername: { [key: number]: string } = {}
 // TODO: if we get multiple tracker calls we should cancel one of the sessionIDs, now they'll clash
-function serverCallMap (dispatch: Dispatch, getState: Function, skipPopups: boolean = false): CallMap {
-  const sessionIDToUsername: { [key: number]: string } = {}
-  const identifyUi = {
-    start: ({username, sessionID, reason}) => {
+function serverCallMap (dispatch: Dispatch, getState: Function, skipPopups: boolean = false, onFinish: ?() => void): incomingCallMapType {
```

Comment:
removed the call map middleware. this makes this look like everything else and also flowtypes better